src/main/java/com/timelogr/enterprise/TimeLogrController.java

```diff
@@ -5,51 +5,55 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-//ALL WEB FILES GO IN THE resources/templates DIRECTORY
+/**
+ * ALL WEB FILES GO IN THE resources/templates DIRECTORY
+ */
 @Controller
 @RequestMapping("/")
 public class TimeLogrController {
-    private TimeLogrServices timeLogrServices;
 
+    private final TimeLogrServices timeLogrServices;
 
     @Autowired
     public TimeLogrController(TimeLogrServices timeLogrServices){
         super();
         this.timeLogrServices = timeLogrServices;
     }
 
-
-
+    /**
+     * STARTS PROGRAM IN THE resources/templates/index.html
+     */
     @RequestMapping("/")
     public String index(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
-        return "index"; //STARTS PROGRAM IN THE resources/templates/index.html
+        model.addAttribute("employee", this.timeLogrServices.getAllEmployees());
+        return "index";
     }
 
     @RequestMapping("/clients")
-    public String Clients(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
+    public String clients(Model model){
+        model.addAttribute("employee", this.timeLogrServices.getAllEmployees());
         return "clients";
     }
 
     @RequestMapping("/settings")
-    public String Settings() {
+    public String settings() {
         return "settings";
     }
 
     @RequestMapping("/newproject")
-    public String NewProject() {
+    public String newProject() {
         return "newproject";
     }
 
     @RequestMapping("/projectdetails")
-    public String projectdetails(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
-        return "projectdetails"; //STARTS PROGRAM IN THE resources/templates/index.html
+    public String projectDetails(Model model){
+        model.addAttribute("employee", this.timeLogrServices.getAllEmployees());
+        return "projectdetails";
     }
-    //pass in project ID
-
-    //@RequestMapping("/logTime")
-    // go to servicestub and call timeLog DAO save
 
+    /*
+    Pass in project ID
+    @RequestMapping("/logTime")
+    Go to service stub and call timeLog DAO save
+    */
```

Comment:
Avoid committing commented-out code to version control, as version control offers better approaches:
1) If you are curious what used to be there, diff commits to see history.
2) If it is an experimental feature, use a feature branch to keep it separate from master.